zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiPlugin.kt

```diff
@@ -40,6 +43,16 @@ class ObjectenApiPlugin(
     @PluginProperty(key = "authenticationPluginConfiguration", secret = false)
     lateinit var authenticationPluginConfiguration: ObjectenApiAuthentication
 
+    @PluginAction(
+        key = "delete-object",
+        title = "Delete object",
+        description = "Delete an object from the Objecten API",
+        activityTypes = [ActivityType.SERVICE_TASK_START]
+    )
+    fun deleteObject(@PluginActionProperty objectUrl: URI): HttpStatus {
```

Comment:
Can you validate if the objectUrl that was provided matches with what the objecten API plugin is configured for? Example: https://github.com/valtimo-platform/valtimo-backend-libraries/blob/920cff453c19926029e1e2bb346a0df3ee5cde9a/zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/DocumentenApiPlugin.kt#L140

Reply:
@mritense I took the liberty of adding a test for it.